next-app/src/routes/+page.svelte

```diff
@@ -31,93 +32,176 @@
 			value: 500_000,
 		},
 	]
+	let items = [
+		{
+			name: 'item 1',
+			prop: 'prop 1',
+			number: 1,
+		},
+		{
+			name: 'it 2',
+			prop: 'property 2',
+			number: 22,
+		},
+		{
+			name: 'itm 3',
+			prop: 'prp 3',
+			number: 333,
+		},
+		{
+			name: 'item 4',
+			prop: 'propty 4',
+			number: 4444,
+		},
+		{
+			name: 'item 5',
+			prop: 'proprty 5',
+			number: 55555,
+		},
+		{
+			name: 'item item item item 6',
+			prop: 'proprty proprty proprty 6',
+			number: 6,
+		},
+		{
+			name: 'item 7',
+			prop: 'property 7',
+			number: 77777777777,
+		},
+	]
 
 	onMount(() => dark_mode = window.matchMedia('(prefers-color-scheme: dark)').matches)
 
 	$: browser && document.documentElement.setAttribute('data-theme', dark_mode ? 'dark' : 'light')
+
+	function sort_items_by(field) {
+		items = items.sort((a, b) => a[field] < b[field] ? 1 : -1)
```

Comment:
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore

Are we targeting ES6?  I'm not sure...

Reply:
I agree, I'd rather avoid it. If we really want some utility these are a really good option since they're one offs without any dependencies. 
https://anguscroll.com/just/

Reply:
I had forgotten about that just lib and do like it but the sort-by's second parm can't handle nested sort out of the box, we'd have to customize the incoming function which is possible of course but at that point is becoming less worth the effort when the code essentially does the same thing (https://github.com/angus-c/just/blob/master/packages/array-sort-by/index.mjs).  I'm certainly open to more discussion.